### DISCUSSION TITLE
WebSocket connection failed

### DISCUSSION THREAD

**Discussion by tklein1801** [supabase/supabase#1805] [Questions] [answered]
I have successfully deployed my application on Netlify. On my localhost demo it all works fine but on my Netlify application the WebSocket connection fails.

I don't receive any error message so I really don't know what do to with this issue .

**Comment by inian** [comment, marked as answer]
Hey @tklein1801, this is an issue on our side and the fix is being rolled out right now. It should be fully deployed in an hour.

**Reply by inian** [reply]
If you give your project ref, I can prioritise rolling out the fix for you.

**Reply by tklein1801** [reply]
Thank you, my project ref should be **fcdxyfclgwhbwughkszz**.

**Reply by inian** [reply]
Hey, your project has been upgraded which should fix this issue

**Reply by tklein1801** [reply]
Yep, thank you very much. The problem is solved :D
